src/Materials/PBR/babylon.pbrBaseMaterial.ts

```diff
@@ -487,13 +492,55 @@
             this._useLogarithmicDepth = value && this.getScene().getEngine().getCaps().fragmentDepthSupported;
         }
 
+        /**
+         * Gets the current transparency mode.
+         */
+        @serialize()
+        public get transparencyMode(): Nullable<number> {
+            return this._transparencyMode;
+        }
+
+        /**
+         * Sets the transparency mode of the material.
+         */
+        public set transparencyMode(value: Nullable<number>) {
+            if (this._transparencyMode === value) {
+                return;
+            }
+
+            this._transparencyMode = value;
+
+            this._forceAlphaTest = (value === PBRMaterial.PBRMATERIAL_ALPHATESTANDBLEND);
+
+            this._markAllSubMeshesAsTexturesDirty();
+        }
+
+        /**
+         * Specifies whether or not the meshes using this material should be rendered in alpha blend mode.
+         */
         public needAlphaBlending(): boolean {
+            return false;

```

Comment:
for consistency I would prefer having this one not defined at all and rely on parent

Reply:
Can't do that if we want transparencyMode to override it unless I also check transparencyMode here.

Reply:
Perhaps then we should do it then
I want people to be able to rely on this function if they need it

Reply:
Agreed. Fixed.